create_config.py

```diff
@@ -1,21 +1,24 @@
-from jinja2 import Template
+import os
 import json
+from jinja2 import Template
 
+DEBUG = True
+output_dir = "output_configs"
 
+with open('ConfTemplate.j2', 'r') as fp:
+    loadtemplate = fp.read()
 
-with open('ConfTemplate.txt','r') as fp:
-    loadtemplate=fp.read()
-
-with open('Clients.txt') as json_file:
-    clients=json.load(json_file)
-
-jinjatemplate=Template(loadtemplate)
+with open('Clients.json', 'r') as json_file:
```

Comment:
did it work?

Reply:
It is working for me, I will double check tonight if I find 5 minutes. If you want you can also checkout the pretty_changes branch and try it on your machine